assets/messages.yml

```diff
@@ -28,42 +28,42 @@ variables:
     - project: bds
       value: |-
         *Quick Links*:
-        📓 [Bug Tracker Guidelines|https://aka.ms/MCBugTrackerHelp] -- 💬 [Community Support|https://discord.com/invite/58Sxm23] -- 📧 [Mojang Support|https://aka.ms/Minecraft-Support]
+        📓 [Bug Tracker Guidelines|https://aka.ms/MCBugTrackerHelp] -- 💬 [Community Support|https://discord.com/invite/58Sxm23] -- 📧 [Mojang Support (Technical issues)|https://aka.ms/Minecraft-Support] -- 📧 [Microsoft Support (Account issues)|https://support.xbox.com/contact-us/]
```

Comment:
This was done to distinguish between both support services and I thought it a good idea to link to Microsoft in mji-qul and after each helper message incase it's helpful for the user